backend/api/src/get-mod-reports.ts

```diff
@@ -6,20 +6,21 @@ export const getModReports: APIHandler<'get-mod-reports'> = async () => {
   const pg = createSupabaseDirectClient()
 
   const reports = await pg.many<ModReport>(`
-    SELECT 
+    select 
       mr.*, 
-      cc.data->>'content' AS comment_content, 
-      c.question AS contract_question, 
-      c.slug AS contract_slug, 
-      creator.username AS creator_username,
-      owner.data->>'username' AS owner_username,
-      owner.data->>'avatarUrl' AS owner_avatarUrl, 
-      (owner.data->>'isBannedFromPosting')::boolean AS owner_isBannedFromPosting
-    FROM mod_reports mr
-    JOIN contract_comments cc ON cc.comment_id = mr.comment_id
-    JOIN contracts c ON c.id = mr.contract_id
-    JOIN users creator ON creator.id = c.creator_id
-    JOIN users owner ON owner.id = mr.user_id
+      cc.data->>'content' as comment_content, 
```

Comment:
Oh, I think you might have wanted only one '>':  cc.data->'content'. That would return the jsonb data instead of a string. I'm not 100% sure, but it could have solved your issue.

Reply:
Nice this allowed me to use our existing function parseJsonContentToText, good catch